src/kaocha/watch.clj

```diff
@@ -86,6 +87,81 @@
         patterns (map #(.getPathMatcher fs (str "glob:" %)) patterns)]
     (some #(.matches % path) patterns)))
 
+
+(defn convert 
+  "Converts a Git-style ignore pattern into the equivalent pattern that Java PathMatcher uses."
+  [pattern] 
+  (let [cleaned  (-> pattern 
+                   ;If a Git pattern has a trailing space, it should be stripped (unless escaped)
+                   ;Example: 'src/test ' => 'src/test'
```

Comment:
Please use `;;` instead of `;`

Reply:
My current commenting plugin (commentary) doesn't distinguish between whole-line and part-of-line comments. Happily, there appears to be another solid one that does (tcomment), so I'll try that one.